index.html

```diff
@@ -14,7 +14,11 @@
 
   <h1 id="title">freeCodeCamp Survey Form</h1>
   <p id="description">Thank you for taking the time to help us improve the platform</p>
+<!-- <div>
+  <h1>
 
+  </h1>
+</div> -->
```

Comment:
Please don't leave unused code in the code base, this makes the code not clean and long